spec/unit/client/redis/apns/notification_spec.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: US-ASCII
+
+require "unit_spec_helper"
+
+describe Rpush::Client::Redis::Apns::Notification do
+  it_behaves_like 'Rpush::Client::Apns::Notification'
+
+  it "should default the sound to 'default'" do
+    notification = described_class.new
+    expect(notification.sound).to eq('default')
+  end
+
+  # skipping these tests because data= for redis doesn't merge existing data
```

Comment:
redis notification does not override the data= method to merge in the existing keys.  I'm not sure if we want that (there was a bug opened on that: #156)